### ISSUE TITLE
Enquiries about choice of SSL methods

### ISSUE THREAD

**Issue by jinhong-ni** [LTH14/rcg#3]
Dear Authors,

Thanks for open-sourcing your great work! I have a few questions regarding the choice of the SSL image encoder in your pipeline.

In the paper, the experiments were mainly conducted with contrastive learning methods like MoCo-v3. These methods without doubt offer compact and informative representation space for conditioning. The denoising process is operated over the representation space. A decoder (pixel generator) is then separately trained to map the representation back to pixel space.

My concern arises from this aspect: masked image modeling (MIM) methods like MAE train a decoder jointly with the image encoder, thus it seems to naturally fit better for representation conditioning and pixel reconstruction. Is there any reason that you pick contrastive learning methods over MIM methods? What are the main benefits gained from employing contrastive learning rather than MIM?

Thanks in advance for your answer. Please correct me if I made any mistake about your work.

**Comment by LTH14** [comment]
Thanks for your interest! Our choice of the pre-trained encoder is mainly based on its linear probing performance, as the encoder is fixed for RCG. We also tried an encoder pre-trained with DeiT, and it also gave good results. MIM-based methods such as MAE provide great fine-tuning performance, but their linear probing performance lags behind contrastive learning. As a result, we tried MAE-based image encoder, but the performance is not as good as contrastive-based encoder.

**Comment by LTH14** [comment]
1. We do not fine-tune the MAE decoder, as it only possesses reconstruction ability but not generation ability (the reconstructive process of MAE is deterministic). When ablating different encoders, our pixel generator is set to MAGE-B.
2. As shown in the MAE and Moco paper, on ImageNet-1K, ViT-B trained with MAE achieves 68.0% linear probing accuracy, while ViT-B trained with Moco v3 achieves 76.7% linear probing accuracy.
